### PR TITLE
Add caretaker project

### PR DESCRIPTION
**Pull request by YuCJ**
All information to develop are documented in the project's README.md
https://github.com/YuCJ/static-fe-boilerplate/tree/caretaker/caretaker

### CODE REVIEW

**Comment by YuCJ** [comment]
Wait for all materials to be ready. And need to fix several problems reported. (See kanban)

File: caretaker/src/components/background.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  `}
+  ${mq.desktopOnly`
+    flex: 0 0 32px;
+    background-image: url(${props => (props.reversed ? desktopBgRight : desktopBgLeft)});
+  `}
+  ${mq.hdAbove`
+    flex: 0 0 40px;
+    background-image: url(${props => (props.reversed ? hdBgRight : hdBgLeft)});
+  `}
+`
```
Comment (taylrj) [inline caretaker/src/components/background.js]: It seems that adding `z-index` value bigger than `40`(which is the `z-index` value of `slidesContainerFocus`) will fix the disappeared opening column issue.

Comment (taylrj) [reply caretaker/src/components/background.js]: My fault! Modify the `z-index` will lead to `PopUp` button being hidden by `Column`.
I created a [PR](https://github.com/YuCJ/static-fe-boilerplate/pull/1) to fixe this issue.

Comment (YuCJ) [reply caretaker/src/components/background.js]: Thanks for solving the problem. 🙏  I've merged the PR.

**Pull request merged**